sros2/scripts/dds_sql_to_sros2_policy.py

```diff
@@ -0,0 +1,143 @@
+#! /usr/bin/env python3
+
+import argparse
+import os
+import pandas as pd
```

Comment:
Note for later: needed to install this dep by hand, we should list it in the `package.xml` ([python3-pandas](https://github.com/ros/rosdistro/blob/a63054565aa50062f25d4fc558e5555b989587ef/rosdep/python.yaml#L5088)) when this is ready for prime time

Reply:
See: 0b27a81